web/src/common.ts

```diff
@@ -162,4 +162,11 @@ export function setup_password_visibility_toggle(
         e.stopPropagation();
         toggle_password_visibility(password_field_id, password_selector, tippy_tooltips);
     });
+    $(password_selector).on("keyup", (e) => {
```

Comment:
Please add a new test case to the `common.test.js` file.

Reply:
I think you might be able to address this without writing a new test by reusing the existing handler and changing

`const handler = $(password_selector).get_on_handler("click")`

to look up the `keyup click` handler instead in the existing test.

https://zulip.readthedocs.io/en/latest/testing/testing-with-node.html will be helpful background.

The only other question is why you're using `keyup`; usually `keydown` or `keypress` is more normal.

Reply:
> The only other question is why you're using keyup; usually keydown or keypress is more normal.

**keypress** is deprecated as by this doc my [mdn](https://developer.mozilla.org/en-US/docs/Web/API/Element/keypress_event)

**keyup** is more convenient for me as a user has to release the key first (ordinary key press) and not hold it as holding the key would fire **keydown** unnecessarily over and over until the user release the key (rise their finger up)

Reply:
That’s how keyboards work, and users know that, so they aren’t going to be holding the key for that long. Use `keydown`.

Or better, use an actual `<button>` element, for which `Enter` automatically fires the `click` event.